src/programmers/internal/check_string_tags.ts

```diff
@@ -5,14 +5,18 @@ import { IdentifierFactory } from "../../factories/IdentifierFactory";
 import { IMetadataTag } from "../../metadata/IMetadataTag";
 
 import { FunctionImporter } from "../helpers/FunctionImporeter";
+import { ITagFormatValidator } from "../helpers/ITagFormatValidator";
 import { check_length } from "./check_length";
 
 export const check_string_tags =
     (importer: FunctionImporter) =>
     (input: ts.Expression, tagList: IMetadataTag[]) => {
         const conditions: ts.Expression[] = [];
         for (const tag of tagList)
-            if (tag.kind === "format")
+            if (
+                tag.kind === "format" &&
+                ITagFormatValidator.isCustomFormat(tag) === false
```

Comment:
This TS check can also be skipped as far as I could understand if the value is not one of the known formats that have a built-in validator